yarn.lock

```diff
@@ -12518,12 +12518,7 @@ image-size@^1.0.0:
   dependencies:
     queue "6.0.2"
 
-immer@^9.0.16:
```

Comment:
deduplication. RTK's immer was not the same as we get from `import {produce} from 'immer'`.